pkgs/ffigen/example/ffinative/pubspec.yaml

```diff
@@ -5,7 +5,7 @@
 name: ffinative_example
 
 environment:
-  sdk: '>=3.2.0 <4.0.0'
+  sdk: '>=3.3.0-dev <4.0.0'
```

Comment:
`3.3.0-252.0.dev`

We probably want to mention the actual lower bound here, in case anyone has a locally built SDK of a previous dev version.

Ditto all the other pubspecs.